test/util/http.js

```diff
@@ -415,7 +415,7 @@ class MockHttp {
   }
 
   respondWithData(callback) {
-    return this.respond(() => ({ status: 200, data: callback() }));
+    return this.respond(() => ({ status: 200, data: callback(), headers: new Map([['date', new Date()]]) }));
```

Comment:
Instead of adding this here, could you add it in `_http()` (or maybe `respond()`)? Only if it's easy. I think it makes sense to always return the header, whether it's after `respondWithData()`, `respondWithSuccess()`, or `respondWithProblem()`.